src/components/ImportForm/utils/auth-utils.ts

```diff
@@ -24,6 +24,7 @@ import { useNamespace } from '../../../utils/namespace-context-utils';
 export const useAccessCheck = (
   source: string,
   dependency,
+  needToTrim?,
```

Comment:
Either we always trim inside this utility function, or we rely on the caller to have trimmed the value. But it's unnecessary to parameterize this utility function with the ability to trim or not trim.

Reply:
updated